tests/testbase.py

```diff
@@ -8,9 +8,10 @@
 from requre.utils import StorageMode
 
 
-def network_connection_avalilable():
+def network_connection_available():
     sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-    if sock.connect_ex(("example.com", 80)) == 0:
+    # example.com
+    if sock.connect_ex(("93.184.216.34", 80)) == 0:
```

Comment:
Can we have some constant for that?

And isn't there some better way of checking the connection? (Without a need of hardcoding anything?)
But since it's test-only, I am ok with the fix.

Reply:
not sure if there is better way, it is important to check some global resource, e.g. instead of check some local IP or some network cards setup, because it may still means that globar resources are not available

Or maybe we can also catch this mentioned exception for DNS resolving issue and result it into "no network available"

But exactly, it is test issue so no spending much time to find an elegant solution :-) and also it is why I've added there comment that this is resolved example.com address

Reply:
OK. Can you please use some constant so we don't need to check the name of this IP when going through the code?